packages/react/src/Button/Button.module.css

```diff
@@ -11,31 +11,60 @@
   cursor: pointer;
   border: none;
   background: none;
+  display: inline-grid;
+  grid-template-areas: 'leadingIcon text trailingIcon';
```

Comment:
It doesn't look like we reference these grid areas anywhere, are these needed?

Reply:
I removed the grid regions in the end, thanks for pointing this out. Removed 👍